src/_includes/cloud/note-docker-ssl-not-private-connection.md

```diff
@@ -0,0 +1,2 @@
+{:.bs-callout-info}
+If you see the `Your connection is not private` error message while using an HTTPS connection, click **Advanced**, then click the **Proceed to magento2.docker (unsafe)** link. If you use Google Chrome and there is no **Advanced** button, then type `thisisunsafe` to bypass the security warnings.
```

Comment:
Please also add `curl` option to bypass warning

Reply:
added